Node-DC-EIS-cluster/controllers/loader.js

```diff
@@ -171,6 +171,19 @@ exports.initDb = function initDB(req, res) {
   var e_count=0;
   var zipcode_current_index=0;
   var lastname_current_index=0;
+  var original_lname_len=lnames.length;
+  if (lnames.length < lastnamecount) {
+    var new_namecount = lastnamecount - lnames.length;
+    var idx = 0;
+    for (var ii=0; ii < new_namecount; ii++) {
+      var new_name = lnames[idx]+ii.toString();
```

Comment:
@egalli @priyankasprakash Even though JS converts number to strings automatically, I would rather have a clean use of API. Use of Number.toString() API is a clean way to use it.